app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
`Dwarf` class should inherit from the `ABC` class as well, since it does not implement all abstract methods of a `Player` class, thus being also abstract.